dibi/drivers/DibiOracleDriver.php

```diff
@@ -97,9 +151,50 @@ public function disconnect()
 	public function query($sql)
 	{
 		$res = oci_parse($this->connection, $sql);
+		if ($this->binds) {
+			set_error_handler(function ($errno, $errstr, $errfile, $errline, $errcontext) use ($sql) {
```

Comment:
You're right, this slipped, I'll fix it. oci_bind_by_name returns boolean which tells you absolutely nothing about the error. In addition it throws warnings when something is not right, that's why I introduced custom error handler.